lblod/ember-rdfa-editor
# Wrapping selection in rdfa nodes

### Overview
Add ability to wrap a selection in a content/literal or resource node.

##### connected issues and PRs:
Part of https://binnenland.atlassian.net/browse/GN-4579

### Setup
In the editable nodes PoC.

### How to test/reproduce
If there is no selection, nothing new appears.
Select any content in the editor and use one of 'Wrap with literal' or 'wrap with resource' from the sidebar. Adding a resource shows a modal as it's necessary to set the uri to use for the resource.

### Challenges/uncertainties
Prosemirror API around selections and commands.

### Checks PR readiness
